### PR TITLE
[Merged by Bors] - [CU-2mmppm6] lease-period-prolongator

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/2mmppm6

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2mmppm6 Create a script for parachain lease period prolongation](https://app.clickup.com/t/2mmppm6)

**Review by KaiserKarel — APPROVED** [review]

File: scripts/lease-period-prolongator/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# lease-period-prolongator
+
```
Comment (Martin-Kiss) [inline scripts/lease-period-prolongator/README.md:2]: Nit: A short description of the functionality would be appreciated.

**Review by Martin-Kiss — APPROVED** [review]

File: scripts/lease-period-prolongator/README.md (hunk 1 above)
Comment (andor0) [reply scripts/lease-period-prolongator/README.md:2]: Added.

**Review by xonoxitron — APPROVED** [review]

**Review by Martin-Kiss — APPROVED** [review]

**Pull request closed without merging**